app/subscriber/src/features/my-reports/view/components/ReportSections.tsx

```diff
@@ -96,50 +95,75 @@ export const ReportContentForm: React.FC<IReportContentFormProps> = ({ disabled
     <div className="sections">
       <DragDropContext onDragEnd={handleDrop}>
         {values.sections.map((section, index) => {
-          const label = getBlockName(section);
           return (
             <Section
               key={section.id}
-              icon={getSectionIcon(section)}
+              icon={<SectionIcon type={section.settings.sectionType} />}
               open={true}
               label={
                 <Row>
-                  {label}
-                  {section.settings.label !== label ? `: ${section.settings.label}` : ''}
-                </Row>
-              }
-              actions={
-                <Row>
-                  {section.settings.sectionType === ReportSectionTypeName.Content && !disabled && (
-                    <Row gap="1rem">
-                      {/* <Action disabled={isSubmitting} icon={<FaFileExcel />} title="Export to Excel" /> */}
-                      <Action
-                        disabled={isSubmitting || disabled}
-                        icon={!showForm ? <FaPen /> : <FaCheck />}
-                        title="edit"
-                        onClick={() => setShowForm(!showForm)}
-                      />
-                      <Action
-                        disabled={isSubmitting || disabled}
-                        icon={<FaPlus />}
-                        title="Add custom content"
-                        onClick={() => addStory(instance?.id, section.name)}
-                      />
-                    </Row>
-                  )}
+                  <SectionLabel section={section} showIcon={false} />
                 </Row>
               }
+              actions={({ open }) => {
+                return (
+                  <Row gap="1rem">
+                    {/* <Action disabled={isSubmitting} icon={<FaFileExcel />} title="Export to Excel" /> */}
```

Comment:
Might be intentional as well but just in case, commented line here

Reply:
Yes it's something in the design we need to add.  Exporting a section to Excel.  Currently we only support the whole report export.